justusCoding/bartender_LiftOff_Project
# adding code from Spring Initializr

New Spring Boot project from Spring Initializr, using Java 11 and Spring Boot 2.7.13, which everyone should be set up for from our previous projects. From my understanding, to use the latest non-snapshot Spring Boot (3.1.1) we would all need to use at least Java 17. If we want to go that route, I'm happy to reconfigure.
To Test: 
1) Launch Gradle bootRun. Project should build successfully and an empty webpage ("whitelabel error page") should be accessible at localhost:8080
